pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -48,6 +48,11 @@ in stdenv.mkDerivation rec {
     substituteInPlace tools/hid2hci.rules \
       --replace /sbin/udevadm ${systemd}/bin/udevadm \
       --replace "hid2hci " "$out/lib/udev/hid2hci "
+  '' + lib.optionalString doCheck ''
```

Comment:
Less conditionals are preferred for patches to see them break in all conditions:

```suggestion
```